pkg/parser/model/ddl.go

```diff
@@ -782,6 +782,56 @@ func (job *Job) DecodeArgs(args ...interface{}) error {
 	return nil
 }
 
+// InfoJSON is works like MarshalJSON, but is made to have human friendly output, similar to String()
+//
+// - A string for things like Type, State and SchemaState instead of a number
+// - A selection of fields
+func (job *Job) InfoJSON() ([]byte, error) {
+	jobInfo := struct {
```

Comment:
How to make sure it's up-to-date with the model.job?

Reply:
Adding a field to `model.Job` won't automatically add it to `InfoJSON()` as these are not a strict 1-to-1 mapping.